src/cell.rs

```diff
@@ -0,0 +1,24 @@
+use rand::{thread_rng, Rng};
+
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub struct Pos {
+    pub x: i32,
+    pub y: i32,
+}
+
+#[derive(Clone, PartialEq, Debug)]
+pub struct Cell {
+    pub pos: Pos,
+    pub is_dead: bool,
+}
```

Comment:
Because you're storing them in a matrix, you don't need to store the cell's position. Then you can represent this as an enum to avoid having untagged booleans:

```rust
pub enum CellState {
    Dead,
    Alive,
}
```